### ISSUE TITLE
How to use a larger dataset to build the model.

### ISSUE THREAD

**Issue by SherlockHua1995** [darpan-jain/crowd-counting-using-tensorflow#1]
Hello, thanks for your code.  I am working on this field too and I want to build a project that counting people under a surveillance camera.
How to use a large dataset to train the model?
Although you have mentioned in readme , I didn't found the LabelImg .
Any help would be appreciated.

**Comment by darpan-jain** [comment]
Hey @SherlockHua1995, here's the link for LabelImg (https://github.com/tzutalin/labelImg)
This tool is pretty straightforward to use and will give you the coordinates of bounding boxes as XML files for each image, which can be converted to a csv and then into a TFR format. 
The csv and TFR conversion code is available in the official Tensorflow repo. 
You can also have look at these links for the same:
1. https://pythonprogramming.net/creating-tfrecord-files-tensorflow-object-detection-api-tutorial/
2. https://stackoverflow.com/questions/41402332/tensorflow-create-a-tfrecords-file-from-csv

Hope this helps!

**Comment by SherlockHua1995** [comment]
Thanks for your reply.
Is the train set in your repository made by yourself ?
And what size of the training set is sufficient for a accurate model to count the people ?
yours~

**Comment by darpan-jain** [comment]
Hey! Yes the train set is created by me using LabelImg.
The size of the training set depends more your use case. You could try will about a 100 annotated images and add images as per the required accuracy. 
Thanks!
